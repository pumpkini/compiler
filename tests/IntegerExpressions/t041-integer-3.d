int main() {
    int x;
    x = 5;
    Print(x);
    x = -8;
    Print(x);
    x = 5858;
    Print(x);
    x = 306088;
    Print(x);
    x = -382728;
    Print(-382728);
}
