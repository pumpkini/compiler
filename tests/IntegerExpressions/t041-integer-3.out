5
-8
5858
306088
-382728
